openwisp_utils/admin_theme/admin.py

```diff
@@ -18,6 +23,72 @@ class OpenwispAdminSite(admin.AdminSite):
     )
     enable_nav_sidebar = False
 
+    def login(self, *args, **kwargs):
+        from django.http import HttpResponseRedirect
+        from django.urls import reverse
```

Comment:
why these imports here and not at the top?

Reply:
I have moved them to the top. In Django's source code, some imports were done in a similar way like for [`get_urls`](https://github.com/django/django/blob/e0a46367df8b17905f1c78f5c86f88d21c0f2b4d/django/contrib/admin/sites.py#L241)